### PR TITLE
Add Terraform config for logging resource

### PR DESCRIPTION
**Pull request by azhao155**
Add Terraform config for logging resource

### CODE REVIEW

File: google/resource-snippets/logging-v2/test_alternatives.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# assumes that the following projects are created and kubectl context is set to a cluster in KRM_PROJECT
+TF_PROJECT_ID=[TF_PROJECT_ID]
+DM_PROJECT_ID=[DM_PROJECT_ID]
+
+gcloud auth application-default login
+
+# Create DM resources
+gcloud config set project $DM_PROJECT_ID
+gcloud services enable deploymentmanager.googleapis.com --project $DM_PROJECT_ID
+gcloud services enable cloudkms.googleapis.com --project $DM_PROJECT_ID
```
Comment (paulmoon) [inline google/resource-snippets/logging-v2/test_alternatives.sh:12]: This should be logging.googleapis.com. Same with line 18

**Review by paulmoon — APPROVED** [review]
For the next `test_alternatives.sh`, could you please take a look at https://github.com/GoogleCloudPlatform/deploymentmanager-samples/blob/master/google/resource-snippets/pubsub-v1/test_alternatives.sh? I made a few improvements over the "old style" (detailed in #627).

File: google/resource-snippets/logging-v2/test_alternatives.sh (hunk 1 above)
Comment (azhao155) [reply google/resource-snippets/logging-v2/test_alternatives.sh]: updated

File: google/resource-snippets/logging-v2/test_alternatives.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
+set -e
+
+# assumes that the following projects are created and kubectl context is set to a cluster in KRM_PROJECT
```
[the remark this answers is not in the document]
Comment (azhao155) [reply google/resource-snippets/logging-v2/test_alternatives.sh]: updated

**Pull request merged**